refinery/data_set_manager/search_indexes.py

```diff
@@ -30,6 +31,7 @@ class NodeIndex(indexes.SearchIndex, indexes.Indexable):
     uuid = indexes.CharField(model_attr='uuid')
     study_uuid = indexes.CharField(model_attr='study__uuid')
     assay_uuid = indexes.CharField(model_attr='assay__uuid', null=True)
+    data_set_uuid = indexes.CharField(null=True)
```

Comment:
Note that there is no `model_attr`... Wondering if schema.xmls should be checked in, if they can be generated from this? There's a redundancy here.